From 0908344fae3edda3372ee03820ea30ebcfe8980e Mon Sep 17 00:00:00 2001
From: Bo Maryniuk <bo@suse.de>
Date: Thu, 13 Sep 2018 12:00:55 +0200
Subject: [PATCH] Retire MD5 checksum for pkg mgmt plugins

Use SHA256 algorithm for zyppnotify plugin

Remove an empty line
---
 scripts/suse/yum/plugins/yumnotify.py         | 2 +-
 scripts/suse/zypper/plugins/commit/zyppnotify | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/suse/yum/plugins/yumnotify.py b/scripts/suse/yum/plugins/yumnotify.py
index 268e1e9531..dd2485c886 100644
--- a/scripts/suse/yum/plugins/yumnotify.py
+++ b/scripts/suse/yum/plugins/yumnotify.py
@@ -32,7 +32,7 @@ def _get_checksum():
     Returns:
         hexdigest
     """
-    digest = hashlib.md5()
+    digest = hashlib.sha256()
     with open(RPM_PATH, "rb") as rpm_db_fh:
         while True:
             buff = rpm_db_fh.read(0x1000)
diff --git a/scripts/suse/zypper/plugins/commit/zyppnotify b/scripts/suse/zypper/plugins/commit/zyppnotify
index 268298b108..b64badb119 100755
--- a/scripts/suse/zypper/plugins/commit/zyppnotify
+++ b/scripts/suse/zypper/plugins/commit/zyppnotify
@@ -35,7 +35,7 @@ class DriftDetector(Plugin):
         Returns:
             hexdigest
         '''
-        digest = hashlib.md5()
+        digest = hashlib.sha256()
         with open(self.rpm_path, "rb") as rpm_db_fh:
             while True:
                 buff = rpm_db_fh.read(0x1000)
-- 
2.20.1


